api/src/main/resources/api/service_openapi.yaml

```diff
@@ -92,6 +181,24 @@ components:
         boolVal:
           type: boolean
 
+    DataType:
+      description: Enum of data types
+      type: string
+      enum: ['INT64', 'STRING', 'BOOL']
+
+    Entity:
+      type: object
+      description: A relational logical entity supported by an Underlay.
+      properties:
+        name:
```

Comment:
Probably not for this PR: I'm wondering whether entity name and display name will always be the same, or if we'll need to split them out. I guess it will largely depend on how dynamic the frontend is.

Given you're including this in the API now, we will likely need to enforce some restrictions around them (length, characters)

Reply:
Yeah. I can see us having a display name or description for rendering, but having a short meaningful entity name instead of an opaque id seems useful.
I'll add entity name restrictions in a follow up PR.